src/components/ProfileHeader.jsx

```diff
@@ -0,0 +1,87 @@
+import User1 from "../assets/user/user1.svg";
+import { DotsThree } from "@phosphor-icons/react/dist/ssr";
+
+const ProfileHeader = ({ user }) => {
+  const totalPosts = user.recipes?.length + user.reviews?.length;
```

Comment:
Can you add a test when recipes, reviews, followers and following equal 0? I think it might break